src/strategies/call-block-logs/extractor.mjs

```diff
@@ -76,15 +76,8 @@ export function update(message) {
   if (message?.method == "eth_blockNumber") {
     let endBlockNumber = message.metadata.end;
     const latestBlockNumber = parseInt(message.results);
-    if (
```

Comment:
Please exclude this change from the PR. And my preference in general is always to neatly separate concerns. This makes the entire PR very hard to approve quickly.